### DISCUSSION TITLE
Independent validators: introduce yourselves!

### DISCUSSION THREAD

**Discussion by sorpaas** [neatcoin/neatcoin#12] [General]
If you are an independent validator looking for nominators, please feel free to introduce yourself here!

We're also working on more plans to give independent validators more exposures. Please stay tuned!

**Comment by ilgio** [comment]
**ilgio**
I am a graphic designer and I offer hardware and software support. I've gained experience in cryptocurrencies and system engineering related to Linux, Windows, Mac, AS400. I am a developer with good experience with HTML5, CSS3, JS, AJAX, Python, PHP, MySQL.
Expert validator in kusama, polkadot, chainx, edgeware, phala

**Comment by redpenguin-validator** [comment]
redpenguin

I am an enthusiast crypto-coin user and miner since 2014. I started with PoW minining, with a small GPU/FPGA/ASIC mining farm, currently minining mainly PoS coins, loving in particular NPoS and running validators in the polkadot ecosystem (kusama, polkadot, edgeware, chainx). Last, but not least, I got extensive experience in server maintenance, on GNU/Linux and Windows, with some coding experience in VB.Net, C#, C++, PHP, Python and Objective C++ and recently JS.

**Comment by dragonstake** [comment]
[DragonStake.io](https://dragonstake.io)

**DragonStake** is an enterprise fully dedicated to Proof of Stake networks. 
We operate an enterprise-grade infrastructure that is robust and secure. Downtime is not an option for us.

We have been running validators on most important networks since their inception. 

- Cosmos
- Polkadot
- Kusama
- Avalanche
- Kava
- Sentinel

We currently accrued hundreds of millions dollars in stake  on all those networks. 
We focus on **security, reliability** and a **delegators education**.

We participated and won the uptime award at the mythic hackaton Game of Stakes (Cosmos network 2017-2018). We also participated on all past Polkadot Testnets since PoC1 until nowadays MainNet, where we have one (and sometimes two :-) active validators.

We are also responsible of a great Spanish speaking community

**Comment by marco69v** [comment]
here I am,
old fashioned computer worker looking for a way to retire earlier :D 
jokes apart. into crypto world since only 2 year but fashinated and with the wants to do something. Let me try to be a validator. I will not dissapoint you.

**Comment by staking4all** [comment]
🙋‍♀️ Hey all 

👩🏻‍💻 we are a validator that is active in polkadot ecosystem

🥩 We offer staking services on several substrate chains but also other ecosystems 

🌐 Can find more info on our [website](https://www.staking4all.org/)

🗳feel free to delegate to us on Neatcoin ecosystem

🗣 always available on telegram

**Comment by maxinmilan78** [comment]
**MAXINMILAN**
My first experience in the crypto world was mining kulupu; I follow the developments of the polkadot ecosystem. I am a devops engineer with a lot of experience in enterprise environment on Unix/Linux systems.
I would like to use my working experience as Neatcoin validator.

**Comment by dakk** [comment]
dakkk
Opensource software developer, professional staking since 2016 in 10 different chains.

https://github.com/dakk

**Comment by hirishh** [comment]
Hello,

I'm hirish, I'm an opensource full-stack developer involved in crypto since 2012. I'm also a Ledger and Trezor developer. You can find all my contribution on my github: https://github.com/hirishh/

I'm an experienced validator currently running multiple nodes on Polkadot, Kusama and Subsocial

**Comment by gregorst3** [comment]
Also known as Gregorst, I am a young crypto enthusiast. I've started in 2015 by mining the first ethereum block. Then besides pow, I've been part of dpos communities, gaining day by day more knowledges, learned the basics of primary languages codes such as C++ and Python. I'm daily following Polkadot and related projects (Kusama, Plasm, ChainX,Polkadot,Moonriver) with the COSMOON project. Currently, I have a part time job as system engineer.

**Comment by Ondin777** [comment]
Hi guys, I'm Ondin, validator and delegate in some important cryptographic projects, which I believe may have high potential.

**Comment by openbitlab2** [comment]
**Openbitlab** is a professional validator focused on Proof of Stake networks, with block production activities on Kusama, StaFi, Lisk, Mina, and multi-million $ delegated.

**Comment by PolkachuIntern** [comment]
Hey **Polkachu** here. We are a professional team that run validators in many PoS blockchains. Our validators are active in many Substrate-based chains such as Polkadot, Kusama, Polkadex, and Sora. Now we are starting on Neacoin. We are also actively validating on Tendermint-based chains such as Juno Network, Kava, Sifchian, BitCanna, KiChain, and Comdex. Overall, delegators/nominators have trusted us with close to $100M in crypto assets. You can find more information about us on our website: [https://polkachu.com](https://polkachu.com/).

We are active in the Discord channels of these projects, and actively participate in the governance. We open-source all validator deployment scripts to encourage more validators to join the networks so the networks can decentralize. For example, this repo is for the deployment of Polkadot/Kusama validators: [https://github.com/polkachu/polkadot-validator](https://github.com/polkachu/polkadot-validator). And this repo is for other Substrate-based chains (including Neatcoin): [https://github.com/polkachu/substrate-validator](https://github.com/polkachu/substrate-validator).

Happy to be here!

**Comment by zaketac** [comment]
We are **ITA Stakers** a group of experienced Italian developers and sys admins, who are making their way into the cosmos and polkadot ecosystem bottom up. 
As of today we are active as Validator on many of the Cosmos chains, to mention some: Juno, Osmosis, Sentinel, BitSong, Desmos, BitCanna, Lum. For us validating is not only "run and forget", but we always try as our best contributing to the project in many ways. 
We have validated kusama and we have a phala stakepool
